app/enrichment/get-scheme.js

```diff
@@ -7,7 +7,9 @@ const getScheme = async (sourceSystem) => {
     return cachedScheme
   }
   const scheme = await getSchemeFromDb(sourceSystem)
```

Comment:
Need unit tests for this file

Reply:
I've added local integration tests as this is very dependent on implementation of external libraries.  Sequelize in particular does weird things with some data types.  So I'd be happier using the actual implementations.

Reply:
Makes sense -- Sequelize is a mystery sometimes